tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7712,6 +7712,9 @@ def warn_hlsl_incorrect_bind_semantic: Warning< // Is an error in fxc in some ca
   "invalid register specification, expected %0 binding">;
 def err_hlsl_incorrect_bind_semantic: Error< // Is an error in fxc in some cases, but is sometimes ignored
   "invalid register specification, expected %0 binding">;
+def warn_hlsl_semantic_attribute_position_misuse_hint: Warning<
+  "'POSITION' is not considered equal to SV_Position but is treated as a user-defined semantic attribute">,
```

Comment:
optional: I think a slightly more idiomatic clang phrasing might be: `'POSITION' is a user-defined semantic; did you mean 'SV_Position'?`

Reply:
Yes, way better message!